src/pages/Dashboard/Components/TimeEntryForm/TimeEntryForm.js

```diff
@@ -7,7 +7,7 @@ import Select from 'react-select';
 import secToMin from 'sec-to-min';
 import moment from 'moment';
 
-import { createTimeEntry } from '../../Actions/index';
+import { createTimeEntry } from '../../../../store/actions';
```

Comment:
Mmm, por alguna razon me escupe:

`Module not found: Error: Can't resolve 'src/store/actions' in '/home/martin/workspace/js/workshop-react/src/pages/Dashboard/Components/TimeEntryForm'
 @ ./src/pages/Dashboard/Components/TimeEntryForm/TimeEntryForm.js 33:15-43`